node/node.go

```diff
@@ -78,7 +78,7 @@ func NewNode(genDoc *genesis.Genesis, conf *config.Config,
 	}
 
 	http := http.NewServer(conf.HTTP)
-	grpc := grpc.NewServer(conf.GRPC, state, sync, consMgr)
+	grpc := grpc.NewServer(conf.GRPC, state, store, sync, consMgr)
```

Comment:
The reason we defined [Facade](https://github.com/pactus-project/pactus/blob/main/state/facade.go) is to encapsulated the store.
So please avoid it. instead define a function in state.Facade to get the Public Key